Utils/Dataflow/060_upload2virtuoso/uploadTTL.sh

```diff
@@ -69,7 +69,8 @@ OPTIONS:
 }
 
 upload_files () {
-
+  EOProcess=""
+  
```

Comment:
`EOProcess` is explicitly defined in both `upload_files` and `upload_stream`. How user value is supposed to be used?